packages/ses/src/intrinsics.js

```diff
@@ -100,7 +101,15 @@ export const makeIntrinsicsCollector = () => {
       }
       const namePrototype = permit.prototype;
       if (!namePrototype) {
-        throw TypeError(`${name}.prototype property not permitted`);
+        cauterizeProperty(
+          intrinsic,
+          'prototype',
+          false,
+          `${name}.prototype`,
+          console, // TODO should be a proper Reporter
```

Comment:
@kriskowal , please advise on how to arrange to pass a proper Reporter here.

Reply:
@kriskowal I threaded a proper reporter to here in the obvious way. But now I have the same mystery in compartment-shim.js